### ISSUE TITLE
Crash on startup beta v1.3.5.56

### ISSUE THREAD

**Issue by DeadSix27** [robertdroptables-mcmods/JEIDsIntegration#5]
>Critical problem: mixins.jeidsi.dimstack.json:mixins.MixinChunkPrimer target net.minecraft.world.chunk.ChunkPrimer was already transformed.

crash log: https://gist.github.com/DeadSix27/76f3378d68f0cc0a0f1e3b008156e157

**Comment by sam-kirby** [comment]
Could I have an export of your modpack please?

**Comment by sam-kirby** [comment]
Alternatively a copy of debug.log may give some clues as to why this is happening.

**Comment by DeadSix27** [comment]
Very heavy pack, expect lag on a slower system: _(although it doesn't even get past the splash anyway. so guess it doesn't matter how heavy it is)_
https://github.com/DeadSix27/various_stuff/releases/download/0.0/mc12pack-1.0.0.zip

EDIT: I will supply debug.log later

**Comment by sam-kirby** [comment]
No crash for me, took basically the entire hour to load but played fairly smoothly once it had.

**Comment by DeadSix27** [comment]
@sk2048 Yep, the loading is brutal, hence why I often just keep it in the background (the loaded game). 
But otherwise it runs pretty well on my system.

Odd that you got no crash, here is the debug log: https://gist.githubusercontent.com/DeadSix27/f2cd95dfb16bea1d028af0a435a313c4/raw/f55fbb7bf5dd872b255a64ec575c7419116a2f6f/debuglog

By the way, the pack had the old JEIDSI.. Did you update it? .. I just exported my "working" pack and forgot to also include the newer JEIDSI, which the working one doesn't have.

**Comment by WenXin20** [comment]
I got the same crash on game launch almost immediately, something to do with Mixins as well. I'm also not using Sponge Forge or Vertically Stacked Dimensions.
Forge v14.23.5.2838
JEIDSI v1.3.5.56
[Crash Report](https://paste.dimdev.org/uwoyavuliz.mccrash)
[Full Log](https://gist.github.com/WenXin20/da6eea92eca3eaaa925763dee36d59fb)

**Comment by sam-kirby** [comment]
I don't think I can fix this unfortunately. Someone is causing the ChunkPrimer class to be loaded really early in the launch sequence. Before I'm able to easily determine if VSD is loaded.

I'll probably have to drop VSD support for the time being. v1.3.5 has been archived, and you should revert to using v1.3.4.

**Comment by DeadSix27** [comment]
Can't it be resolved on that persons side? Make him load it later?

**Comment by sam-kirby** [comment]
I am unaware of a way to determine who triggers the loading of a specific class. As a result I'd need to test against around 250 mods. Something I don't have time for at the minute just to find out who is doing it. Then I'd need to work out why they're doing it then before making a judgement as to whether I am, in fact, adding a transformer too late, or if they are loading it too early.
